test.py

```
@@ -0,0 +1,2 @@
+def test_function(some_argument):
+  print(arg2*3)
```

Ruff

Undefined name `arg2`

(F821)